README.md

```diff
@@ -8,7 +8,8 @@ This project helps you to remove containers/networks/volumes/images by given fil
 
         $ ./bin/moby-ryuk -p 8080
         $ # You can also run it with Docker
-        $ docker run -v /var/run/docker.sock:/var/run/docker.sock -p 8080:8080 quay.io/testcontainers/ryuk
+        $ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -p 8080:8080 testcontainers/ryuk
+        $ docker run --rm -v "//./pipe/docker_engine://./pipe/docker_engine" -p 8080:8080 testcontainers/ryuk
```

Comment:
```suggestion
        $ # Example for Windows
        $ docker run --rm -v "//./pipe/docker_engine://./pipe/docker_engine" -p 8080:8080 testcontainers/ryuk
```

Reply:
Quotes are not necessary, maybe just `docker run --rm -v //./pipe/docker_engine://./pipe/docker_engine -p 8080:8080 testcontainers/ryuk`?

Reply:
Not sure about the quotes. As far as I remember, the `:` triggered some strange behaviour in a shell on Windows. In which shell did you check the examples? I don't remember if I used PowerShell or ConEmu/Git Bash.

Reply:
Mounting the Unix socket would fail for these shells too. I've tested it on Windows PowerShell and CMD, on macOS Z shell and fish.